CHANGES.txt

```diff
@@ -6,6 +6,7 @@ Unreleased
 ==========
 
 - Fix installation by adjusting dependencies
+- Add support for MongoDB 2 and 3
```

Comment:
and 4?

Reply:
MongoDB 4 has been supported beforehand already.